practice-two/src/index.html

```diff
@@ -75,7 +75,60 @@ <h3 class="banner-section__title">We’re Bring It To The New Level</h3>
   <!--END HEADER-->
   <main>
     <!-- START ADVANTAGES SECTION -->
-    <section></section>
+
+    <section class="advantages-section">
```

Comment:
I suggest you should use class advantages here to create a block element, instead of start a block element from container